modules/apim_gateway/templates/carbon-home/bin/wso2server.sh.erb

```diff
@@ -283,14 +285,21 @@ if [ -z "$JVM_MEM_OPTS" ]; then
    java_version=$("$JAVACMD" -version 2>&1 | awk -F '"' '/version/ {print $2}')
    JVM_MEM_OPTS="-Xms<%=@jvmxms%> -Xmx<%=@jvmxmx%>"
    if [ "$java_version" \< "1.8" ]; then
-      JVM_MEM_OPTS="$JVM_MEM_OPTS -XX:MaxPermSize=<%=@jvmxms%>"
+      JVM_MEM_OPTS="$JVM_MEM_OPTS -XX:MaxPermSize=256m"
```

Comment:
Why remove the parameterization here?

Reply:
Fixed https://github.com/wso2/puppet-apim/pull/166/commits/64782d063234228db982a6a0e2376ea2514189fa